api/v1beta1/cinderbackup_types.go

```diff
@@ -146,5 +146,5 @@ func init() {
 
 // IsReady - returns true if service is ready to serve requests
 func (instance CinderBackup) IsReady() bool {
-	return instance.Status.ReadyCount >= 1
+	return instance.Status.ReadyCount >= 1 || instance.Spec.Replicas == 0
```

Comment:
Should we change this to be?:

```
  return (instance.Status.ReadyCount >= 1 && instance.Spec.Replicas > 0) || (instance.Spec.Replicas == instance.Status.ReadyCount)
```

Because I think it's wrong to say we are ready when we change from replica 1 to replica 0 when we still have the replica up and running, right?

Reply:
will update the pr as discussed on slack